src/System.Text.Utf8/tests/TypeConstraintsTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Reflection;
+using Xunit;
+
+namespace System.Text.Utf8.Tests
+{
+    public class TypeConstraintsTests
+    {
+        [Fact]
+        public void Utf8StringIsAStruct()
+        {
+            var utf8String = "anyString"u8;
```

Comment:
not sure if it makes sense, that would add another abstraction layer which is completely redundant and not really helpful when debugging